routes/api.php

```diff
@@ -131,11 +138,16 @@
 Route::middleware('admin.dependency')->group(function () {
 
     // Initial altitudes and sids
-    Route::delete('sid/{id}', 'SidController@deleteSid')
-        ->where('sid', 'd+');
-    Route::put('sid', 'SidController@createSid');
-    Route::put('sid/{id}', 'SidController@updateSid')
-        ->where('sid', 'd+');
+    Route::middleware('dependency.update:DEPENDENCY_PRENOTE,DEPENDENCY_INITIAL_ALTITUDES,DEPENDENCY_SID_HANDOFF')
+        ->group(
+            function () {
```

Comment:
We on a version for an arrow function?

Reply:
At the moment we still support PHP 7.3, so we can't do that yet :(

Reply:
sad times